### ISSUE TITLE
Any special reasons for using id sampling in dataloader

### ISSUE THREAD

**Issue by Cold-Winter** [AliaksandrSiarohin/first-order-model#431]
Great work! Thanks for releasing the code again.
I notice that you are using id sampling in dataloader to ensure one batch have different persons. Why did you do this, why not just shuffle all the videos?

Best,
Yandong

**Comment by AliaksandrSiarohin** [comment]
The dataset is pretty imbalanced, so one identity could have ~100 videos, while other could have 1. But to be honest I implement this from the very beginning and did not check if it works without it.

**Comment by Cold-Winter** [comment]
Thanks for the quick response. I trained different experiments with the same hyper-parameters and find that they all get very different faces in various quality. Some of them even failed. Do you observe the same issues?

**Comment by AliaksandrSiarohin** [comment]
That is weird usually I get more or less the same results. At least in the terms of loss.
